types/next/index.d.ts

```diff
@@ -102,51 +145,41 @@ declare namespace next {
             query?: QueryStringMapObject,
             parsedUrl?: UrlLike
         ): Promise<void>;
-        renderError(
-            err: any,
+        renderToHTML(
             req: http.IncomingMessage,
             res: http.ServerResponse,
             pathname: string,
             query?: QueryStringMapObject
-        ): Promise<void>;
-        render404(
-            req: http.IncomingMessage,
-            res: http.ServerResponse,
-            parsedUrl: UrlLike
-        ): Promise<void>;
-        renderToHTML(
+        ): Promise<string>;
+        renderError(
+            err: any,
             req: http.IncomingMessage,
             res: http.ServerResponse,
             pathname: string,
             query?: QueryStringMapObject
-        ): Promise<string>;
+        ): Promise<void>;
         renderErrorToHTML(
             err: any,
             req: http.IncomingMessage,
             res: http.ServerResponse,
             pathname: string,
             query?: QueryStringMapObject
         ): Promise<string>;
+        render404(
+            req: http.IncomingMessage,
+            res: http.ServerResponse,
+            parsedUrl?: UrlLike
+        ): Promise<void>;
```

Comment:
Just re-ordered the `render*` methods to match their order in https://github.com/zeit/next.js/blob/6.1.1/server/index.js. No changes here.